modules/kg-schemas/src/main/resources/vocabs/nexus-core.ttl

```diff
@@ -0,0 +1,106 @@
+@prefix : <https://bbp-nexus.epfl.ch/vocabs/nexus/core/v0.0.1/> .
```

Comment:
Any specific reason we're using `ttl` as the format instead of `json-ld`?

Reply:
yeah....